compiler/scopelang/print.ml

```diff
@@ -98,6 +98,24 @@ let scope ?(debug = false) ctx fmt (name, decl) =
              SubScopeName.format_t subscope_name Print.punctuation "]"))
     decl.scope_decl_rules
 
+let print_topdef ctx ppf name (e, ty) =
+  Format.pp_open_vbox ppf 2;
+  let () =
+    Format.pp_open_hovbox ppf 2;
+    Print.keyword ppf "let";
+    Format.pp_print_space ppf ();
+    TopdefName.format_t ppf name;
```

Comment:
Why not use regular `@[` `Format` syntax?

Reply:
I actually find this style more readable when the format string is mostly a concatenation: mentally matching the `%a` with the correct parameters is a pain. Things are also made worse with `ocamlformat` which doesn't allow us to keep a clear pairing between the printers and their arguments.

The `let ()` is here just to visualise the boundaries of the box (I'd have used `begin/end` but `ocamlformat` scraps them).

And it's not that much more verbose when even symbols need a printer for color.